### PR TITLE
⚒ Add greenkeeper support for lockfiles

### PR DESCRIPTION
**Pull request by chrishelgert**

### CODE REVIEW

**Comment by danielbayerlein** [comment]
https://github.com/greenkeeperio/greenkeeper-lockfile#setup

File: .travis.yml [hunk 1]
```diff
@@ -10,5 +9,13 @@ cache:
...
+before_install: yarn global add greenkeeper-lockfile@1
+before_script: greenkeeper-lockfile-update
+after_script: greenkeeper-lockfile-upload
+
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+
+env:
+  matrix:
+    secure: BlMNcXA9uc9vBZulaYSC2+0AslBYb/IeucZO27srRUaen6hick1L/WfWXs29XUb7oWq+fqOfE/+9fDRLrV0fQUvBZWe+T7r/NDTHOGvzd++u9ACjPJj1ALKbSZOn0GeX887sxTroqaJnyvvt4+EPG4e14tkY2cHL4IIs/u1W4Z4nTOpNqfHg2T0Ayse6YnUT+5jZ84JJ7CvxjU7mwcTyQLScsdGT9GNw7bC+35LNUaT454xSkv2feJOeVy2lRVemnuJ5kGVRJN/BVPq6lhmT6qnLhz9EEXNZTuZJJD773CI4VIICc06gwPZLbJTT6BdiAqjeU0S7FQCHtOQhiAAofzvMzZv4qcOWSOPptw77mWlbSeN6pvvRvV2aK5qEXJyqHTNxjX8H2hq/e7Bzi9bC/TshuOrpbZh3ZDAcADQaAEfqJa4UDEpXEd5D18t+BFLfH2MlIMwSqyYFPv32C9ttqtMXuYdrw/vD38tjY50bSFlOzJVf0DJmmX0nvYsGXdcOQaxe3Gcgkv4ExWR3yT9uuy4mFO9cAegwWF3d0gUGuZPfR1PF8e6Az/AwIl94vTovQPCTBKCLCWCyqA9KQj3M2cgeBNrRjKgv0NvftUZKCbBeVIqoX/yctWPfMQzEjGGH854AvnIpWyEgwKrQzlZGVcO04PGyqynzwtxSyU/Asrs=
```
Comment (danielbayerlein) [inline .travis.yml]: Is the access token generated with your personal account?

Comment (chrishelgert) [reply .travis.yml]: yes i can not create an token with the orga - maybe we should think about it later (maybe an bot-account?)

Comment (danielbayerlein) [reply .travis.yml]: Yes, I think a bot account would make sense.

**Pull request merged**